LinuxBootImageGenerator.py

```diff
@@ -1798,7 +1798,8 @@ def __copyFiles2partitison(self,diagnosticOutput = True, partition = None):
     print('#        Any other input: Do not compress the output image                   #')
     print('#                                                                            #')
     print('##############################################################################')
-    _wait_ = input('#              Please type ...                                               #')
+    # _wait_ = input('#              Please type ...                                               #')
+    _wait_  = 'n'
     if _wait_ == 'q' or _wait_ == 'Q':
         sys.exit()
     elif _wait_ =='Y' or _wait_ =='y':
```

Comment:
I think we should get rid of the `_wait_ ` entirely and if we want to compress the image, change to argeparse or something similar.  Then we can have a command line option to compress the final image or not.

Reply:
@fe-tdavis I agree. It would be nice to completely remove the `_wait_` and replace it with argeparse to keep the functionality. However, due to time constraints this would be an enhancement feature since it is not required for our current usage. Please see commit 6c7f2ec and issue 2

Reply:
Yeah, that's a good point.